### PR TITLE
[v10] Update Library for new systemd install 

### PR DESCRIPTION
**Pull request by sagesyr**
Backport for 

https://github.com/gravitational/teleport/pull/15800

### CODE REVIEW

**Review by GavinFrazar — APPROVED** [review]

**Review by strideynet — APPROVED** [review]

File: docs/pages/deploy-a-cluster/open-source.mdx [hunk 1]
```diff
@@ -166,8 +166,16 @@ On your Linux machine, run the following command to start the `teleport` daemon
...
+  like to use for your configuration as part of the command. We recommend the path `/etc/systemd/system/teleport.service` for most use cases:
+
+    ```code
+    $ sudo teleport install systemd -o /etc/systemd/system/teleport.service
+    ```
+  Enable and start the new `teleport` daemon:
+
     ```code
-    $ sudo teleport start
+    $ sudo systemctl enable teleport && sudo systemctl start teleport
```
Comment (alexfornuto) [inline docs/pages/deploy-a-cluster/open-source.mdx:178]: You could simplify this with:

```suggestion
    $ sudo systemctl enable --now teleport
```

File: docs/pages/deploy-a-cluster/teleport-enterprise/getting-started.mdx [hunk 2]
```diff
@@ -124,6 +121,7 @@ ssh_service:
 Start the Teleport service and confirm the service has started. After it has started you can get the Certificate Authority (CA) pin that has been generated.
 
 ```code
+$ sudo systemctl enable teleport
 $ sudo systemctl start teleport
```
Comment (alexfornuto) [inline docs/pages/deploy-a-cluster/teleport-enterprise/getting-started.mdx:125]: As above.

```suggestion
$ sudo systemctl enable --now teleport
```

File: docs/pages/management/admin/daemon.mdx [hunk 3]
```diff
@@ -2,18 +2,16 @@
...
 ---
+On Linux systems in non-containerized environments, we recommend running the `teleport` binary as a daemon using systemd.
+Using a daemon helps to ensure that the `teleport` process can remain running and available regardless of a controlling terminal session or parent process, and opens up additional
+configuration options that allow for better optimization, uptime, and availability.
 
-The Teleport binary is called `teleport`. When you run `teleport start`, the
-Teleport process runs in the foreground. On Linux systems in non-containerized
-environments, we recommend running the `teleport` binary as a daemon using
-systemd.
+This guide will outline best practices for installing, configuring, and starting teleport as a daemon on a Linux host using systemd as of Teleport `v10.1.14`.
```
Comment (alexfornuto) [inline docs/pages/management/admin/daemon.mdx:9]:
```suggestion
This guide will outline best practices for installing, configuring, and starting Teleport `v10.1.14` or later as a daemon on a Linux host using systemd.
```

File: docs/pages/deploy-a-cluster/open-source.mdx (hunk 1 above)
Comment (alexfornuto) [reply docs/pages/deploy-a-cluster/open-source.mdx]: Moving to a new branch

**Review by alexfornuto — APPROVED** [review]

**Pull request merged**